tests/functional/storageclass/test_new_sc_rbd_replica2_3.py

```
@@ -170,7 +171,16 @@ def test_new_sc_new_rbd_pool(
             f"Cluster used space with replica size {replica}, "
             f"compression mode {compression}={cluster_used_space}"
         )
-        if not erasure_coded:
+        if erasure_coded:
+            data_pool = sc_obj.get().get("parameters").get("dataPool")
+            if data_pool:
+                ec_opt = get_ceph_pool_property(data_pool, "allow_ec_optimizations")
+                log.info(f"EC pool '{data_pool}' allow_ec_optimizations: {ec_opt}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)